openml-lightgbm-module/lightgbm-lib/pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ The copyright of this file belongs to Feedzai. The file cannot be
+  ~ reproduced in whole or in part, stored in a retrieval system,
+  ~ transmitted in any form, or by any means electronic, mechanical,
+  ~ photocopying, or otherwise, without the prior permission of the owner.
+  ~
+  ~ (c) 2020 Feedzai, Strictly Confidential
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.feedzai.openml.lightgbm</groupId>
+    <artifactId>lightgbm-lib</artifactId>
+    <version>2.3.190</version>
+
+    <packaging>jar</packaging>
+    <url>https://github.com/feedzai/make-lightgbm</url>
+
+    <properties>
```

Comment:
Pull this back to the lightgbm-module pom so we can reuse it and lightgbmlib.version is in sync in the two child poms (this one and the provider itself).

Reply:
will do

Reply:
ok, I see why the properties are here because I can't heritage from the parent. Maven does allow it

Reply:
So can we move it to the lgbm-module?